pages/sequential.jsx

```diff
@@ -21,8 +21,35 @@ import Address from '../components/address-input.jsx';
 import Email from '../components/email-input.jsx';
 import PrivacyPolicy from '../components/privacy-policy-input.jsx';
 
-var SingleForm = React.createClass({
-  mixins: [require('react-intl').IntlMixin, require('../mixins/form.jsx')],
+import {FormattedMessage, IntlMixin, FormattedNumber} from 'react-intl';
+
+var DonateButton = React.createClass({
```

Comment:
This is just the display for the submit button. This display makes it a donate button.